guppylang/tys/ty.py

```diff
@@ -423,15 +423,21 @@ def transform(self, transformer: Transformer) -> "Type":
         )
 
 
-# We define the `Type` type as a union of all `TypeBase` subclasses defined above. This
-# models an algebraic data type and enables exhaustiveness checking in pattern matches
-# etc.
-# Note that this might become obsolete in case the `@sealed` decorator is added:
-#  * https://peps.python.org/pep-0622/#sealed-classes-as-algebraic-data-types
-#  * https://github.com/johnthagen/sealed-typing-pep
+#: The type of parametrized Guppy types.
 ParametrizedType: TypeAlias = FunctionType | TupleType | SumType | OpaqueType
```

Comment:
We have to add these `#:` docstrings to all type aliases and also do `from __future__ import annotations` everywhere to make it work. I'll leave that to another PR